package simplejavaprogram;

public class MyOwnException {
	public static void main(String[] args) {

		for (int i = 5; i >= 0; i--) {
			System.out.println(10/i);
		}
		System.out.println("After for loop...");
	}
}
